ios/SitumLocationWrapper.m

```diff
@@ -282,6 +282,28 @@ - (SITLocationRequest *) dictToLocationRequest: (NSDictionary *) dict {
             outdoorLocationOptions.enableOutdoorPositions = [[options valueForKey: @"enableOutdoorPositions"] boolValue];
         }
         
```

Comment:
Instead of the string literala like "updateInterval" I recommend using constants